### DISCUSSION TITLE
Language Methods

### DISCUSSION THREAD

**Discussion by shary998** [florisboard/florisboard#1808] [Q&A] [answered]
Does Floris Board get Spanish Version? Like Predictions in Spanish and Spell checker for Spanish Language??

**Comment by patrickgold** [comment, marked as answer]
Spell checker can already be done in Spanish, word suggestions will be available when Latin word suggestions are implemented, so currently in the 0.4.0 dev cycle, which comes after 0.3.16.

**Reply by shary998** [reply]
Thanks
